custom/icds/location_reassignment/parser.py

```diff
@@ -140,12 +140,12 @@ def _invalid_row(self, row, location_type_code):
         new_site_code = row.get(NEW_SITE_CODE_COLUMN)
         invalid = False
         if old_site_code in self.requested_transitions:
-            if self.requested_transitions.get(old_site_code) != operation:
+            if self.requested_transitions.get(old_site_code) != SPLIT_OPERATION or operation != SPLIT_OPERATION:
```

Comment:
this now ensures that there is a repetition only in case of specific operations.
Only, for a split operation should the old site code repeat and
only, for a merge operation should the new site code repeat.